lib/withOnyx.js

```diff
@@ -48,16 +49,22 @@ export default function (mapOnyxToState) {
 
             componentDidMount() {
                 // Subscribe each of the state properties to the proper Onyx key
-                _.each(mapOnyxToState, (mapping, propertyName) => {
+                const mapKeys = Object.keys(mapOnyxToState);
+                for (let i = 0; i < mapKeys.length; i++) {
+                    const propertyName = mapKeys[i];
+                    const mapping = mapOnyxToState[propertyName];
```

Comment:
From what I have read, `for...in` performance is supposedly not as fast as a regular `for` loop. But I wonder if it makes a huge difference in practice. Since this code is really "low level" I'm mostly going with the idea that since we are calling these functions very often in the front end code that any small difference could yield a large one at scale.